internal/cli/down.go

```diff
@@ -48,6 +48,8 @@ func (c *downCmd) down(ctx context.Context, downDeps DownDeps) error {
 		return err
 	}
 
+	tlr.DockerPruneSettings.Disabled = true // don't docker prune on tilt down
```

Comment:
i wonder if we should do the opposite: DockerPruneSettings.Enabled, which default to false unless the Tiltfile has run and this is a 'tilt up'

Reply:
are you saying that the default for the USER should be that DockerPrune is disabled unless they explicitly enable it, or just that in the CODE we should default to `disabled` unless it's a `tilt up`?

Reply:
this is just a comment about how to organize the code

Reply:
k, done!